<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadAndPrintClass</class>
 <widget class="QMainWindow" name="ReadAndPrintClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1605</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReadAndPrint</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ImageViewer">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>701</width>
      <height>721</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="openImageBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="outputText">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>671</width>
      <height>721</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nextImageBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>750</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevImageBtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>750</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QLabel" name="numOfImages">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>750</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Set Crop Region</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Position X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Position Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="posXTextBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="PosYTextBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>100</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>130</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="widthTextBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="heightTextBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="readandprint.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_openImageBtn_clicked()</slot>
  <slot>on_processBtn_clicked()</slot>
  <slot>on_prevImageBtn_clicked()</slot>
  <slot>on_nextImageBtn_clicked()</slot>
 </slots>
</ui>
